src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let cashCount = 1;
+
+  return (a) => (b) => {
```

Comment:
This way is more readable.
```suggestion
  return (a) => {
    return (b) => {
```